src/promptflow/tests/executor/unittests/_core/test_tools_manager.py

```diff
@@ -176,13 +176,6 @@ def test_collect_package_tools_if_node_source_tool_is_legacy(self):
         assert "promptflow.tools.azure_content_safety.analyze_text" in package_tools.keys()
 
     def test_collect_package_tools_and_connections(self, install_custom_tool_pkg):
```

Comment:
why we do not need reload any longer?

Reply:
Because `importlib.metadata` queries the environment's metadata in real-time and it can access the new installed packages without reloading.